### ISSUE TITLE
Overflow menu Usage lacking some standard sections; also think this is a variant of Button

### ISSUE THREAD

**Issue by mbgower** [carbon-design-system/carbon-website#3121]
A review of the [Overflow menu's](https://ibm.box.com/s/5mfqci3e3wgp0khaxqjqahwewolepim1) and [Style](https://ibm.box.com/s/7lh5waldg9o9tt2o5rc8533a2majbjpy) pages was carried out as part of an accessibility review. Note that there are comments from both Jess Lin and myself.

The largest consideration that came out of the accessibility review is that the Overflow menu is arguably an example of a button menu -- a component lacking in Carbon. It likely makes the most sense as a variant of Button, but there are other possibilities (such as making Button menu a separate component and Overflow menus two of the variants). I list a few considerations in my comments in the Usage PDF.

The Overflow menu Usage page also lacks some standard sections, including Variants, Anatomy and Behaviors. Including these would help assess a number of considerations.
